woocommerce/wc-admin
# Update @types/wordpress__notices version to fix type error

This PR updates ` @types/wordpress__notices` from `1.5.4` to `^3.3.0` for latest `@wordpress/notices`.

I made the changes for ` @types/wordpress__notices` and [the PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58884) was just merged.


### Screenshots

Before:

<img src="https://user-images.githubusercontent.com/4344253/158104112-0264b19e-447b-49d1-bc05-e1fa59cf9d39.png" width="500" >

After:

<img src="https://user-images.githubusercontent.com/4344253/158104123-6fb5f534-3220-4a0d-b1b1-e9bd7ae5050f.png" width="500" >




### Detailed test instructions:

- Open `client/tasks/task-list-item.tsx`, confirm there is a type error of `onClick`.
- Checkout this branch & `pnpm install`
- Confirm the type error disappeared


no changelog
